pantry/common/api_common.py

```diff
@@ -7,3 +8,27 @@ def get_columns(parameters, columns):
             cols.append(columns[col])
 
     return cols if len(cols) > 0 else columns
+
+
+def filter_columns(parameters, query, supported_columns):
```

Comment:
Not really sure what happens in this method to be honest. c > 'gt' etc. I will give you the benefit of the doubt but it might be worth putting a comment or two in the method.